jena-arq/src/main/java/org/apache/jena/sparql/ARQConstants.java

```diff
@@ -292,6 +292,10 @@
     public static final Symbol registryServiceExecutors =
         SystemARQ.allocSymbol("registryServiceExecutors") ;
 
+    /** The service executor library registry key */
```

Comment:
Are two different keys necessary?

Reply:
Very good question. Right now the bulk and non-bulk registries are two separate classes. I need to think about whether they could be combined into a single class.